resources.rc

```diff
@@ -0,0 +1,34 @@
+#define IDR_ICON 101
+IDR_ICON              ICON                      "icon.ico"
+
+#define VER_MAJOR 1
+#define VER_MINOR 4
+#define VER_REVISION 5
+
+
+
+#define INNER_TO_STR(x) #x
+#define TO_STR(x) INNER_TO_STR(x)
+#define VER_STRING TO_STR(VER_MAJOR) "." TO_STR(VER_MINOR) "." TO_STR(VER_REVISION) ".0"
+
+#define VS_VERSION_INFO 0x1L
+
+VS_VERSION_INFO VERSIONINFO
+FILEVERSION     VER_MAJOR,VER_MINOR,VER_REVISION
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904E4"
+        BEGIN
+            VALUE "FileDescription",  "Symphony Screen Share Indicator Frame"
+            VALUE "ProductName",      "Symphony Screen Share Indicator Frame"
+            VALUE "ProductVersion",   VER_STRING
+            VALUE "LegalCopyright",    "Copyright 2020 Symphony OSS"
```

Comment:
@mattias-symphony  I think we should make the year dynamic. Otherwise, we'll lose track of it.

Reply:
It's not easily done though, we'd need to do some tooling to generate a code file. Could be a future improvement